.ci.yaml

```diff
@@ -1201,7 +1201,8 @@ targets:
       - bin/internal/engine.version
       - .ci.yaml
 
-  - name: Linux tool_integration_tests_1_5
+  - name: Linux tool_integration_tests_1_6
+    bringup: true
```

Comment:
We have to mark all of these as bringup because we're renaming the shard?

Reply:
yep